buildpacks/jvm-function-invoker/buildpack.toml

```
@@ -2,7 +2,7 @@ api = "0.8"
 
 [buildpack]
 id = "heroku/jvm-function-invoker"
-version = "0.6.6"
+version = "0.6.7"
```

The bot updates the buildpack version numbers automatically - the `0.6.6` version hasn't been released yet, so is already the new version. The only time we need to manually increment the version is if we want to do a minor/major version bump instead of a patch release.

My usual trick to work out what files updating for changes like this, is to look at a previous bump and copy that. For example:
https://github.com/heroku/buildpacks-jvm/commit/b2d6407dd00e88e43d9aed750c9a8c7d579e6a03

```suggestion
version = "0.6.6"
```